bin/test_daq

```diff
@@ -13,7 +13,7 @@ function delay_finish {
     sleep 10
 }
 
-if [ -n "$TRAVIS" ]; then
+if [ -n "$CI" ]; then
```

Comment:
Where is CI defined?  Is that defined as part of GitHub actions?  If possible, I'd prefer something longer than "CI" (short names are always problematic), but if it's dictated from GitHub then it'll have to do...

Reply:
Yes, it's one of the github actions default env vars. https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables